__tests__/features/end-to-end.feature

```diff
@@ -1,8 +1,8 @@
 Feature: End to end
 
   As a user
-  I want to be able to perform a Home Check
-  So that I can check a home
+  I want to be able to perform a Tenancy and Household Check
+  So that I can check on a tenancy
 
```

Comment:
is this right?

Reply:
that will be changed in one of the subsequent tickets